llvm/lib/Target/Sparc/SparcFrameLowering.cpp

```diff
@@ -97,12 +98,6 @@ void SparcFrameLowering::emitPrologue(MachineFunction &MF,
   // Debug location must be unknown since the first debug location is used
   // to determine the end of the prologue.
   DebugLoc dl;
-  bool NeedsStackRealignment = RegInfo.shouldRealignStack(MF);
```

Comment:
`hasStackRealignment` now also always returns false (there are a couple of uses in this file).